styles.css

```diff
@@ -0,0 +1,1381 @@
+body {
+  font-family: sans-serif, poppins;
+  background: #e5e5e5;
+  position: relative;
+}
+
+@media only screen and (min-width: 1200px) {
+  .grid-layout {
+    display: grid;
+    grid-template-columns: 6% 88% 6%;
+    justify-content: stretch;
+  }
+
+  .grid {
+    margin: 5px;
+    margin-bottom: 142px;
+  }
+
+  .g2 {
+    max-width: 100%;
+    background-color: #fff;
+    border-radius: 20px;
+    display: flex;
+  }
```

Comment:
- [ ] Please I highly recommend your stick to the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1) as much as possible.

_**REQUIRED FIGMA TEMPLATE**_

![Screenshot (87)](https://user-images.githubusercontent.com/78104544/227413311-bb933cd9-ba17-48eb-bbe3-f9900e0255e0.png)


_**YOUR DESIGN**_

![Screenshot (85)](https://user-images.githubusercontent.com/78104544/227413414-5d40edf1-2b31-42ac-823b-b301e8dddbda.png)

Reply:
i followed the pc template and the i made it responsive, that why its like that

Reply:
@braincee

Reply:
i didnt note that pc template were a lot different than mobile  version, till i was almost done. do i need to change it?

Reply:
@gersonhiggins, Hi great work on thinking about responsiveness already in this project 💪 
Therefore this project requires students to build the project using a mobile-first approach (Build a project for mobile version first then start making it responsive for desktop/pc). So please keep in mind that for now, you should stay focused on the mobile version only 📱 (Don't hesitate to tag me @trast00, question, I will answers)

Reply:
@Trast00 okay, yeah i just tried my best. it still has a mobile version so you can check it out